_posts/2015-01-23-drivers.md

```diff
@@ -89,7 +89,7 @@ Drivers should focus solely on being an interface for effects, and usually are l
 
 {% highlight js %}
 run(main, {
-  log: msg$ => { msg$.subscribe(msg => console.log(msg)) }
+  log: msg$ => { msg$.addListener({next: msg => console.log(msg)}) }
```

Comment:
@staltz recently said on Gitter that only next is required. I've not double checked the code though. Thanks for comment.